sdk/src/main/java/me/digi/sdk/api/DMEAPIClient.kt

```diff
@@ -38,6 +38,30 @@ class DMEAPIClient(private val context: Context, private val clientConfig: DMECl
     private val httpClient: Retrofit
     internal val argonService: DMEArgonService
 
+    companion object {
+
+        fun parsedDMEError(
```

Comment:
`parsed` is the derivative noun of the verb `parse`. As the function is doing an action, the wording should use the root verb, so `parseDMEError` please. 🙂

Reply:
Oh nice, fixed :D